docs/docs-ref-autogen/outlook/office.attachmentcontent.yml

```diff
@@ -57,3 +57,33 @@ items:
       return:
         type:
           - MailboxEnums.AttachmentContentFormat
+        description: |-
+
+          #### Examples
+
+          ```javascript
+          var item = Office.context.mailbox.item;
+          var listOfAttachments = [];
+          item.getAttachmentsAsync(callback);
+          function callback(result) {
+              if (result.value.length > 0) {
+                  for (i = 0 ; i < result.value.length ; i++) {
+                      var options = {asyncContext: {type: result.value[i].attachmentType}};
+                      getAttachmentContentAsync(result.value[i].id, options, handleAttachmentsCallback);  
+                  }
+              }
+          }
+
+          function handleAttachmentsCallback(result) {
+              // parse string to be a url, an .eml file, a base64-encoded string, or an .icalendar file
+              if (result.format == Office.MailboxEnums.AttachmentContentFormat.Base64) {
```

Comment:
Not sure if these are intentional or not. I think this one is ok as long as this Base64 thing always returns a non-zero number? (which hopefully it will do)